lib/internal/modules/run_main.js

```diff
@@ -92,10 +94,30 @@ function shouldUseESMLoader(mainPath) {
  */
 function runMainESM(mainPath) {
   const { loadESM } = require('internal/process/esm_loader');
-  const { pathToFileURL } = require('internal/url');
+  const { pathToFileURL, URL } = require('internal/url');
+  const _importMapSpecifier = getOptionValue('--experimental-import-map');
```

Comment:
This underscore prefix is pretty uncommon in Node.js. I prefer not using it.

Reply:
Ah, yeah that is just a habit I have. Is there a more commonly used pattern for variables like this which are "not in their final form yet"? This one is "processed" into the real specifier layer, so I used the same name but with the `_` to indicate that. Happy to follow whatever folks consider more "standard".

Reply:
It's a const, isn't it in its final form? `importMapFlagValue`?

Reply:
I probably phrased that poorly, but that value is not modified but is processed by resolution. Anyway, I like that name and will change to that next I get a chance to work on this.